device-types/Cisco/WS-C6513-E.yaml

```diff
@@ -0,0 +1,47 @@
+---
+manufacturer: Cisco
+model: Catalyst 6513
+slug: ws-c6513
+part_number: WS-C6513
+u_height: 19
+is_full_depth: true
+subdevice_role: parent
+console-ports:
+  - name: con0
+    type: rj-45
+interfaces:
+  - name: FastEthernet1
+    type: 100base-tx
+    mgmt_only: true
+power-ports:
+  - name: PS1-1
+    type: iec-60320-c20
+    maximum_draw: 2800
+  - name: PS1-2
+    type: iec-60320-c20
+    maximum_draw: 2800
+  - name: PS1-3
+    type: iec-60320-c20
+    maximum_draw: 2800
+  - name: PS2-1
+    type: iec-60320-c20
+    maximum_draw: 2800
+  - name: PS2-2
+    type: iec-60320-c20
+    maximum_draw: 2800
+  - name: PS2-3
+    type: iec-60320-c20
+    maximum_draw: 2800
```

Comment:
This should be made modular with appropriate PSU ports depending on the PSU inserted